content/working_with/console/pages/dashboard-page.md

```diff
@@ -14,30 +14,20 @@ The Dashboard page is the landing page when you log into the {{< param cfy_conso
 
 ### Statistics
 
-By default, the page displays usage statistics of the {{< param cfy_manager_name >}}, including the number of:
-
-* [blueprints]({{< relref "working_with/console/widgets/blueprintNum.md" >}})
-* [deployments]({{< relref "working_with/console/widgets/deploymentNum.md" >}})
-* [plugins]({{< relref "working_with/console/widgets/pluginsNum.md" >}})
-* [node instances]({{< relref "working_with/console/widgets/nodesComputeNum.md" >}})
-* [running executions]({{< relref "working_with/console/widgets/executionNum.md" >}})
-
-on this {{< param cfy_manager_name >}}r.
-
+By default, the page displays usage statistics of the {{< param cfy_manager_name >}}, including number of services and number of Kubernetes clusters on this {{< param cfy_manager_name >}}.
+The data is presented using [number of deployments widget]({{< relref "working_with/console/widgets/deploymentNum.md" >}}).
 
 ### Actions
 
 You can also:
 
-* [upload a blueprint]({{< relref "working_with/console/widgets/blueprintUploadButton.md" >}})
-* [create a deployment]({{< relref "working_with/console/widgets/deploymentButton.md" >}})
-* [upload a plugin]({{< relref "working_with/console/widgets/pluginUploadButton.md" >}})
+* [set up a cloud account]({{< relref "working_with/console/widgets/buttonLink.md" >}})
+* [create a service]({{< relref "working_with/console/widgets/serviceButton.md" >}})
+* [create a Kubernetes cluster]({{< relref "working_with/console/widgets/serviceButton.md" >}})
 * go to Getting Started Walkthrough tour with [single button]({{< relref "working_with/console/widgets/buttonLink.md" >}}) click
```

Comment:
It's not there anymore since RD-3566 was implemented ;-) 

(I know the page content is changing very fast, that's unfortunate)

Reply:
I removed it for now. I think adding terraform point, which should replace it, should be done as part of ticket dedicated for covering the terraform related feature.